cumulusci/core/config/BaseGlobalConfig.py

```diff
@@ -23,3 +32,37 @@ def get_project_config(self):
     def create_project(self, project_name, config):
         """ Creates a new project configuration and returns it """
         raise NotImplementedError("Subclasses must provide an implementation")
+
+    @property
+    def config_global_local_path(self):
+        directory = os.path.join(os.path.expanduser("~"), self.config_local_dir)
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+
+        config_path = os.path.join(directory, self.config_filename)
+        if not os.path.isfile(config_path):
+            return None
+
+        return config_path
+
+    @property
+    def config_global_path(self):
+        return os.path.abspath(
+            os.path.join(__location__, "..", "..", self.config_filename)
+        )
+
+    def _load_config(self):
+        """ Loads the local configuration """
+        # load the global config
+        with open(self.config_global_path, "r") as f_config:
+            config = yaml.safe_load(f_config)
+        self.config_global = config
+
+        # Load the local config
+        if self.config_global_local_path:
+            config = yaml.safe_load(open(self.config_global_local_path, "r"))
+            self.config_global_local = config
+
+        self.config = MergedConfig(
+            global_local=self.config_global_local, global_config=self.config_global
+        ).config
```

Comment:
I'd still like to see this as function instead of a class. You're extracting the `config` attribute as soon as it's constructed and it doesn't look like MergedConfig being a subclass of BaseConfig is really giving us anything except for more class hierarchy to figure out.